sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -247,6 +247,8 @@ export default defineComponent({
           studentInfo.disabilityStatus,
         ),
         studentProfileDisabilityStatusValue: studentInfo.disabilityStatus,
+        studentProfileModifiedIndependentStatusValue:
```

Comment:
Out of the PR context. We talked about checking with business on displaying the modified independent status in student application section. Was it captured in any of the tickets?